scripts/publish.js

```diff
@@ -74,7 +86,7 @@ if (/master/.test(currentBranch)) {
     'We are on the master branch. Proceeding to also tag it as latest'
   );
   shell.exec(
-    `docker tag ${DOCKER_HUB_REPOSITORY}:${SALESFORCE_CLI_VERSION} ${DOCKER_HUB_REPOSITORY}:latest`
+    `docker tag ${DOCKER_HUB_REPOSITORY}:${SALESFORCE_CLI_VERSION}-slim ${DOCKER_HUB_REPOSITORY}:latest`
```

Comment:
I followed the example in the Node Docker Repo where there is only one `latest` tag that will contain the layers of the `slim` (original image).
https://hub.docker.com/_/node